README.md

```diff
@@ -239,10 +246,10 @@ If your extension requires one or more of those plugins to be present, make sure
 The Hide Tribe Bar extension requires that The Events Calendar 3.0 or greater is present. This line is the only one needed to specify that:
 
 ```
-$this->add_required_plugin( 'Tribe__Events__Main', '3.0' );
```

Comment:
The whole reason I wanted to do a hide tribe bar example was to showcase a reason why you might want to do a version compare with a very old version, such as 3.0. It helps explain a little bit more about how the framework functions, to any who are interested. I would leave this as `3.0`.